integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -57,24 +59,36 @@ def get_context_data(self, **kwargs):
 
         return context
 
-    def get_unreviewed_pages_context(self):
+    def get_latest_versions(self):
         """
-        Extend context by info on unreviewed pages
+        Collects all the latest page translations of the current region
+
+        :return: all the latest page translations of the current region
+        :rtype: ~django.db.models.query.QuerySet
 
-        :return: Dictionary containing the context for unreviewed pages
-        :rtype: dict
         """
-        unreviewed_pages = (
+        latest_version_ids = (
             PageTranslation.objects.filter(
-                status=status.REVIEW,
-                language__slug=self.request.region.default_language.slug,
                 page__id__in=Subquery(
                     self.request.region.non_archived_pages.values("pk")
                 ),
             )
-            .order_by("page__id", "language__id", "-version")
             .distinct("page__id", "language__id")
-            .all()
+            .values("pk")
```

Comment:
```suggestion
            .values_list("pk", flat=True)
```
I'm sorry for the many additional comments :see_no_evil: 

Apparently, `values()` still keeps the queryset lazy evaluated, which means its result will not be cached, but evaluated again in all the subqueries:
![Screenshot 2023-11-01 at 19-54-52 Integreat Redaktionssystem](https://github.com/digitalfabrik/integreat-cms/assets/16021269/703b3120-3873-4194-ab31-332024b757a7)


So we'd have to convert it to an actual list to make use of the ids:
![Screenshot 2023-11-01 at 19-52-48 Integreat Redaktionssystem](https://github.com/digitalfabrik/integreat-cms/assets/16021269/ef1ab075-73df-483a-b3c8-e3a502aa7881)

Not sure how big the impact really is :thinking: 
In the end, this will actually execute one query more, but I assume the computation time will still be smaller, since the result is reused in 4 other queries...